app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -113,6 +113,21 @@ def pod_wait
     end
   end
 
+  def verify_scanned_image_id
+    metadata = image_inspector_client.fetch_metadata
+    actual = metadata.Id
+    return nil if actual == options[:docker_image_id]
+
+    if metadata.RepoDigests
+      metadata.RepoDigests.each do |repo_digest|
+        return nil if repo_digest == options[:docker_image_id]
+      end
+    end
+
+    "cannot analyze image %s with id %s: detected id was %s" % [
+      options[:image_full_name], options[:docker_image_id][0..11], actual[0..11]]
```

Comment:
@enoodle don't you have to update this being more specific now? (Maybe the check that failed was about the `RepoDigests`) No need for anything fancy... I just feel that this message now is misleading.